core/controlplane/config/templates/cloud-config-worker

```diff
@@ -142,7 +142,8 @@ coreos:
         {{end}}--register-node=true \
         {{if .Experimental.Taints}}--register-schedulable=false \
         {{end}}--allow-privileged=true \
-        --pod-manifest-path=/etc/kubernetes/manifests \
+        {{if .Experimental.NodeStatusUpdateFrequency}}--node-status-update-frequency={{.Experimental.NodeStatusUpdateFrequency}}
```

Comment:
@jeremyd Perhaps reading comments around the `worker.nodePools[]` key in the `cluster.yaml` would help but to be short:

```yaml
worker:
  nodePools:
  - name: myPool1
    nodeStatusUpdateFrequency: <value as you like>
```